src/TeX/EmitTeX.sml

```diff
@@ -269,6 +269,16 @@ local
       case overrides s of
         NONE => string_map (s,sz)
       | SOME r => r
+  fun translate_prime s =
```

Comment:
As  `x'¹⁰'` and similar is possible, it seems like you should check for `sup_1` and `sup_0` too.

Reply:
Agreed